pages/linux/bpftrace.md

```diff
@@ -0,0 +1,28 @@
+# bpftrace
+
+> High-level tracing language for Linux eBPF.
+> More information: <https://github.com/iovisor/bpftrace>.
+
+- Display bpftrace version:
+
+`bpftrace -V`
+
+- List all available probes:
+
+`sudo bpftrace -l`
+
+- Run a one-liner program (e.g syscall count by program):
+
+`sudo bpftrace -e {{'tracepoint:raw_syscalls:sys_enter { @[comm] = count(); }'}}`
+
+- Run a program from a file:
+
+`sudo bpftrace {{path/to/file}}`
+
+- Trace a program by PID:
+
+`sudo bpftrace -e {{'tracepoint:raw_syscalls:sys_enter /pid == {{pid}}/ { @[comm] = count(); }'}}`
```

Comment:
Oh... We should find how to espace the `}` in markdown files for the `{{pid}}` :
![Screenshot_20201016_090729](https://user-images.githubusercontent.com/1536740/96256441-0eebb380-0f8f-11eb-99b1-ae8482ed3bf7.png)

Reply:
@navarroaxel so there's no support for nested `{{ }}`? maybe we should just replace `{{pid}}` for something like `PID`

Reply:
We cannot do much about escaping. Each client may parse commands in a slightly different way, and nested braces support is up to the particular implementation. There is no standard escape syntax. 

In order to get around this, I would suggest one of the following:

1. Remove the outermost `{{` and `}}` braces, and only keep them around `pid`.
2. Remove the braces around `pid` and only keep the outermost ones. Change `pid` to `PID_TO_TRACE`.